riotfile.py

```diff
@@ -2925,6 +2926,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                                     pkgs={
                                         "protobuf": "==3.8.0",
                                     },
+                                    create=True,  # Needed bp Python 3.5 because of namespace packages
```

Comment:
```suggestion
                                    create=True,  # Needed by Python 3.5 because of namespace packages
```

also.... I am not sure I understand this comment, what about namespace packages requires this?

don't we install protobuf normally with `ddtrace` ? (dep of `ddsketch`)

Reply:
This is needed because now riot installs dependencies with a different venv, and the `pth` files end up in that side venv 🙁 . This is why we now need to create this venv. The dependencies of the dev package are fine though, because they get installed in the right place, and we use that environment to run commands.